CHANGELOG.md

```diff
@@ -1,7 +1,66 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
-## 11.0.0-RC.1 – 2021-02-12
+## 11.1.0 – 2021-02-23
+### Added
+- Integrate with Deck to allow posting Deck cards to Talk conversations
+  [#5201](https://github.com/nextcloud/spreed/pull/5201)
+  [#5202](https://github.com/nextcloud/spreed/pull/5202)
+  [#5203](https://github.com/nextcloud/spreed/pull/5203)
+- Allow other apps to register message actions, e.g. Deck can create a Deck card out of a chat message
+  [#5204](https://github.com/nextcloud/spreed/pull/5204)
+- Allow to delete chat messages
+  [#5205](https://github.com/nextcloud/spreed/pull/5205)
+  [#5206](https://github.com/nextcloud/spreed/pull/5206)
+- Add information about callFlags of a conversation to the API so mobile clients can show if it's a audio or video call
+  [#5208](https://github.com/nextcloud/spreed/pull/5208)
+
+### Fixed
+- Prevent loading old messages twice on scroll which could skip some messages
+  [#5209](https://github.com/nextcloud/spreed/pull/5209)
+
+## 11.0.0 – 2021-02-22
+### Added
+- Implement read status for messages including a privacy setting
+  [#4231](https://github.com/nextcloud/spreed/pull/4231)
+- Allow moderators to "open" conversations so users can join themselves
+  [#4706](https://github.com/nextcloud/spreed/pull/4706)
+- Add conversation descriptions
+  [#4546](https://github.com/nextcloud/spreed/pull/4546)
+- Allow pagination for main grid view
+  [#4958](https://github.com/nextcloud/spreed/pull/4958)
+- Allow to send messages again when they failed
+  [#4975](https://github.com/nextcloud/spreed/pull/4975)
+- You can now push to talk/mute with the space key
+  [#4328](https://github.com/nextcloud/spreed/pull/4328)
+- Added support for turns:// protocol
+  [#5087](https://github.com/nextcloud/spreed/pull/5087)
+- Add basic support for Opera
+  [#4974](https://github.com/nextcloud/spreed/pull/4974)
+- Allow resending email invitations
+  [#5052](https://github.com/nextcloud/spreed/pull/5052)
+- Allow to collapse the video strip to focus more on the promoted speaker or screenshare
+  [#4363](https://github.com/nextcloud/spreed/pull/4363)
+- Improve previews of images and allow animation of gifs
+  [#4472](https://github.com/nextcloud/spreed/pull/4472)
+- Allow to "Raise hand" in a call
+  [#4569](https://github.com/nextcloud/spreed/pull/4569)
+- Compatibility with Nextcloud 21
+
+### Changed
+- Bring up the conversation creation-dialog when clicking on a group to prevent accidentally spam
+  [#5062](https://github.com/nextcloud/spreed/pull/5062)
+- User different border color when own message is quoted
+  [#4940](https://github.com/nextcloud/spreed/pull/4940)
+- Move matterbridge settings into the conversation settings dialog
```

Comment:
If this entry is worth mentioning, we should also add an entry for mentioning that many options from the top right menu were moved to that dialog. Currently we don't even have a statement in the changelog that there's this new conversation dialog in the first place.

Reply:
So the reason I added that one is because it completly changes position, from right sidebar having a tab to a dotdotdot-button > Submenu option.
The other settings are more logically moved just one hierarchy down, I gues that can be guessed by the name.